schema/record-package-schema.json

```
@@ -156,12 +156,12 @@
         },
         "compiledRelease": {
           "title": "Compiled release",
-          "description": "This is the latest version of all the contracting data, it has the same schema as an open contracting release.",
+          "description": "The latest value of each field at the time of the record's publication, it has the same schema as an open contracting release.",
           "$ref": "https://standard.open-contracting.org/schema/1__1__5/release-schema.json"
         },
         "versionedRelease": {
           "title": "Versioned release",
-          "description": "This contains the history of all the data in the compiled release.",
+          "description": "A history of changes to each field up to the time of the record's publication.",
```

Align with the wording of `compiledRelease`.

```suggestion
          "description": "The history of values of each field up to the time of the record's publication.",
```